blocks/form/form.js

```diff
@@ -84,7 +86,12 @@ function createOutput(fd) {
   const wrapper = createFieldWrapper(fd);
   const output = document.createElement('output');
   output.name = fd.Name;
-  output.innerText = fd.Value;
+  const displayFormat = fd['Display Format'];
+  if (displayFormat) {
+    output.dataset.displayFormat = displayFormat;
+  }
+  const formatFn = formatFns[displayFormat] || formatFns.identity;
```

Comment:
```suggestion
  const formatFn = formatFns[displayFormat] || (x) => x;
```

this makes it a bit more robust against the identity formatter missing.

Reply:
Resolved.
https://github.com/vdua/nedbank/pull/1/files#diff-14d0faad899dcfcb4e5cd318658f4d58b9e2aa66ebc5912bbe09302339f6a564R147